src/Console/Command/FixCommand.php

```diff
@@ -184,7 +184,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $progressOutput = $showProgress && $stdErr
-            ? new ProcessOutput($stdErr, $this->eventDispatcher)
+            ? new ProcessOutput($stdErr, $this->eventDispatcher, count($resolver->getFinder()))
```

Comment:
The cost depends on the Finder implementation used and on what set it is used, so the numbers provided give some indication but not much IMHO.
It is the same discussion as here https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1408
I'm not in favor (but not a strong opinion) to always count ahead, but as an opt-in I think this looks nice (or provide an opt-out back to the current way of dots).
Happy to hear more opinions though!